### PR TITLE
FCCEUD-28: Describe how to view a rendered preview of a module

### PR DESCRIPTION
**Pull request by theashiot**
Issue ID: FCCEUD-28
Issue Link: https://projects.engineering.redhat.com/browse/FCCEUD-28

* Create module "Previewing modules in pantheon"

### CODE REVIEW

**Comment by emarcusRH** [comment]
looks good to me too

**Comment by adahms** [comment]
@redhataccess/pantheon-developers  - Please review for technical completeness and accuracy.

**Comment by benradey** [comment]
@adahms What you have is correct. For completeness' sake, I mention that you don't *have* to be logged in if you want to view the released version of the doc.

So, if you're not logged in, you can still search for your module, and when you select it from search results, the system will take you straight to the released view of the doc - bypassing its details screen. (There is no ability to view the draft version - or even see whether a draft version exists - if not logged in).

**Comment by benradey** [comment]
@adahms Let me know if you'd like to update the doc with this new information, or if you'd like to merge it in as-is - either way is fine.

**Comment by benradey** [comment]
Oops, looks like @theashiot is the original author here, so tagging him as well.

**Comment by theashiot** [comment]
Thanks @benradey, I think it makes sense to update the doc in this pull request. I'll make the required changes.

best,
ashwin

**Comment by theashiot** [comment]
Hi @benradey,
I've modified the prerequiste to indicate that non-logged in users can see published modules. Do you and @adahms think this is ok or should I explain it in more details?

best,
ashwin

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+include::../attributes.adoc[]
```
Comment (adahms) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc:1]: Remove.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+[id="previewing-modules-in-pantheon_{context}"]
+= Previewing modules in {ProductShortName} as logged-in user
```
Comment (adahms) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc:3]: I'm not sure we need to include 'as logged-in user' in the title here - you've called that out nice and clearly in the text itself, and the general task we're attempting to perform is previewing a module. Being logged in is a detail of that, and if we included it in the title, readers might think there are other procedures for different variations.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
++
+If you are not logged in, you can still see the rendered module if it is already published to the customer portal.
+
+* The module to be previewed has been imported in {ProductShortName}.
+
+.Procedure
+. Navigate to the link:{LinkToSearchPage}[Search page].
+. Click the name of the module you want to preview.
++
+If you don't see the module listed, search for the modules using the search box.
```
Comment (adahms) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc:18]: s/don't/do not/

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc (hunk 1 above)
Comment (theashiot) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc:1]: done

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc (hunk 2 above)
Comment (theashiot) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc:3]: Indeed, changed!

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc (hunk 3 above)
Comment (theashiot) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc:18]: ouch!!

**Comment by theashiot** [comment]
Thanks @adahms for the suggestions, I have incorporated them.

@emarcusRH can you please provide another peer-review (because a significant amount of content has changed since the earlier review)?

many thanks,
ashwin

**Comment by theashiot** [comment]
Thanks @emarcusRH, fixed the typo.

**Comment by theashiot** [comment]
Thanks @benradey,

No more changes from my side. I resolved draft, it is ready to be merged.

best,
ashwin

**Comment by theashiot** [comment]
@redhataccess/pantheon-developers Please merge.

**Pull request merged**